apps/super-app/src/routes/SupportDetailWip.tsx

```diff
@@ -74,14 +85,14 @@ function DescriptionSection(props: DescriptionProps) {
   return (
     <main className="support-description-container">
       <section className="support-description-header">
-        <img src={logo} style={{ flexBasis: '8rem' }} />
+        <img src={logo} style={{ flexBasis: '6rem', width: '1px' }} />
```

Comment:
I am not sure why this changed (probably just better sizing?), but I don't see any issues with this change.

Reply:
Yeah, it seemed like better spacing as I was eyeballing it. The `width: 1px` is there to give an initial width that the flex box can grow later.